Sources/NIOSSL/SwiftCrypto/NIOSSLSecureBytes.swift

```diff
@@ -0,0 +1,375 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2022 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+
+public struct NIOSSLSecureBytes {
```

Comment:
Can we begin by removing all the `public` access qualifiers from this type and then add them in as we need in follow-up PRs? I'd like us to avoid exposing too much public API unnecessarily.